src/components/hwWallet/accountCard.js

```diff
@@ -27,14 +27,18 @@ const AccountCard = ({
       {fromRawLsk(account.balance)}<p>LSK</p>
     </div>
     {isEditMode ?
-      <div className={styles.edit} onClick={(e) => {
+      <div
+        className={styles.edit}
+        onClick={/* istanbul ignore next */(e) => {
```

Comment:
@michaeltomasik do you think you could put the message outside the code, I mean like
```
onClick={(e) => { /* istanbul ignore next */
```

Maybe that works too

Reply:
I tried and it needs to be there so it does not lower test coverage